content/v0.0.1/index.md

```diff
@@ -6,3 +6,288 @@ aliases: ["/en/"]
 # Conventional Logs v0.0.1
 
 ## Summary
+
+The conventional logs specification is an opinionated convention
+aiming to set an industry standard to ensure search-ability
+and enhance security in logs.
+
+Each log entry could be encoded in different formats, `json`, `xml`, or even a custom one.
+Most of the examples in this page will be written as `json`.
+
+A basic log entry should be structured as follows:
+
+---
+
+```
+{
+  "severity": "DEBUG",
```

Comment:
WDYT about adding a property to put the error? most of the times we want to have a log message which is not the same as the error provided by some client but we still want to be able to see the error

we have many cases in go, for example while talking to the database and it returns an unknown error usually we wanna see the error message but just put a default log message

Reply:
Agree with it!
Something like `"error": "nil pointer exception"` would definitely help

Reply:
@guiajlopes added the error field, let me know what you think